rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -540,7 +540,7 @@ public JobResultData jobResult(@HeaderParam("sessionid") String sessionId, @Path
     }
 
     // request:  http://localhost:8080/rest/scheduler/jobs/resultmap?jobsid=102&jobsid=152
-    // response: {"102": {"var1" : "val1", "var2" : "val2"},"152": {}}
+    // response: {"102": {"var1" : "val1", "var2" : "val2"}}
```

Comment:
1) How can one check  if all his job ids have or don't have resultMaps ? this will add in the calling-code a test for all ids sent for this endpoint to see if they have a result or not. I'm not sure that this is the proper way to do this.

2) It is not a good idea to use this endpoint the way it is now with multiple job ids because HTTP/GET calls are limited to 2000 chars. Therefore, if you have 6 figures job id (say 10100, 10101, etc...) and you send a query for 100 jobs, your url will be at least:(`"&jobsid=".length` = 8) 
`(8+6)*100` = 1400 !,
At Komatsu this will break in less than a week 😢 
If you cannot expect an HTTP/GET max length as is the case here they you should do an HTTP/POST

Reply:
Wow, thanks
can you send a link to `limited ti 2000 chars`

Reply:
https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers
I remember that Job Planner (during @paraita's days) had an issue that caused this and made the job planner fail at scale

Reply:
1) Alternatively, the endpoint could return hasResultMap=true/false but in that case, from a pure REST API perspective, the endpoint has no great value. Here it gives the opportunity to receive all in a single call.
2) the intent of this endpoint is to restrict it to a small number of jobids (<= 100).

This is why it is implemented using GET and it matches the workflow automation portal requirement which calls this on less than 100 jobs.

If ever in the future we really need to perform this call on a large number of jobs, then we will probably need to implement it via POST (we can actually have two endpoints with the same path and different methods)